Hello Microverse features

devendra-alt opened the pull request:
Modified  
-README.md
-LICENSE.md

issue #1

dev-en-m committed:
Modified README.md

dev-en-m committed:
Modified LICENSE to MIT LICENSE

dev-en-m committed:
Corrected typos in LICENSE.md

aamirkhan2478 commented on README.md:
```diff
@@ -55,10 +157,28 @@ Contributions, issues, and feature requests are welcome!
+
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
- [ ] As mentioned by the previous reviewer #1 :point_left: kindly add a valid LICENSE. Your `MIT` link is not working so kindly fix it :+1:  Check below :point_down: my suggestions :slightly_smiling_face: 

```suggestion
This project is [MIT](./LICENSE.md) licensed.
```

aamirkhan2478 reviewed (changes requested):
Hi @devendra-alt, 

![](https://media3.giphy.com/media/RcZlXVml8oKtoGQeqe/giphy.gif?cid=ecf05e47cw5pjd1gapdzk1t3vp0n73caujoqxw3vkcu8ssng&rid=giphy.gif&ct=g)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linter passed

 ## Required Changes ♻️
- [x] Your PR title and summary are not descriptive kindly edit them with descriptive PR title and summary. Check below :point_down:  the `example` of descriptive PR title and summary :slightly_smiling_face: 

**Example**
![219520522-f0a8c7ba-b272-4a26-81c7-c233913f9c1e](https://user-images.githubusercontent.com/45226848/234373917-0e7a5d21-fc52-4d8e-a17b-cec96d0e169d.png)

- [x] Please remove `.hintrc` from the `.gitignore` file because this is a `hint linter config` file and this should not be ignored so kindly fix :+1: 
- [x] Please create a `test.md` file with the text "This file should be ignored by git". 
- [x] Please edit your branch name as well because your branch name should be any action or feature eg. `add-heading` so kindly rename it. If you don't know how to rename branch name then watch this [video](https://www.youtube.com/watch?v=LYJ81OKXH9c&t=232s&pp=ygUhaG93IHRvIGVkaXQgYnJhbmNoIG5hbWUgaW4gZ2l0aHVi) carefully :slightly_smiling_face: 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dev-en-m committed:
Updated README.md

dev-en-m committed:
Updated .gitignore

dev-en-m committed:
Created test.md

dev-en-m committed:
Updated .gitignore

devendra-alt commented:
As Suggested I have done changes as follows

-Removed .hintx file from .gitignore 
-Added test.md with the text "This file should be ignored by git"
-I have tried to rename the branch but as I have noticed once I rename this branch it will automatically remove this pull request, so that's why I am leaving it as it is for now as "code-review".

as per your suggestion, I will always remember this tip to give meaningful branch names in my future projects.

Thank You for your support 🫡!

dev-en-m committed:
Changed typos in .gitignore

bobb-Rob reviewed (approved):
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @devendra-alt , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/2c67944c14f5c74b9b607df18d73efb0c423dd148b3e10cb75b445a00220332e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
